packages/flutter/lib/src/gestures/events.dart

```diff
@@ -334,8 +334,9 @@ class PointerAddedEvent extends PointerEvent {
     int device = 0,
     Offset position = Offset.zero,
     bool obscured = false,
-    double pressureMin = 1.0,
-    double pressureMax = 1.0,
+    double pressure = 1.0,
```

Comment:
PointerAdded shouldn't have a pressure, there's no interaction by the point this event is sent.